src/main/java/com/codehusky/huskyui/states/action/Action.java

```diff
@@ -31,10 +32,14 @@
  */
 public class Action {
 
-    @Nonnull private final StateContainer container;
-    @Nonnull private final Player observer;
-    @Nonnull private final ActionType type;
-    @Nonnull private final String goalState;
+    @Nonnull
```

Comment:
Sorry, was just auto formatting and the default style guide formats like that. Wasn't intentional